scripts/development/start-dev.sh

```diff
@@ -284,6 +284,7 @@ else
     fi
 fi
 
+
 # Check if the docker-compose command exists. Newer Docker versions come with compose command along the docker. (docker compose)
```

Comment:
Do we want to keep the check there (allow the docker-compose command)? Or just use the "docker compose" as a default one?

Reply:
Leave it here for now. `docker-compose` still works